automated/linux/ethernet/test-eth3.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+source "../test_format.sh"
+
+TEST_NAME="Test ETH-3"
+
+#Worker Host IP_ADDRESS!
+IP_ATE="192.168.168.73"
+
+echo "$TXT_Section1"
+echo "$Test_Start $TEST_NAME"
+echo "Connecting with ATE IP-Address: $IP_ATE"
+echo "$TXT_Section1"
+
+echo "$TXT_Section2"
+echo "$TEST_NAME: 1/2 - Start"
+echo "$TXT_Section2"
+iperf3 -t 1800 -4 -c $IP_ATE -t 10
+
+if [ $? -eq 0 ]
```

Comment:
when $? return non-zero, it means only the command executes failed, nothing more. It doesn't mean the bandwidth is as much as we expected, also we might want to check the latency, loss, etc. 

In general I would think the $? can be just used to do the first check of the success of command executing, testcase fail or pass should be on other criteria we expected.

Reply:
I think we should use/refer this facility from Linaro : https://github.com/Linaro/test-definitions/blob/master/automated/utils/send-to-lava.sh

Reply:
Btw, for the iperf tests, there is already testcase(at least test script) from Linaro, we can maybe use it, or do some analysis of it whether it can be used by us directly.
https://github.com/Linaro/test-definitions/blob/master/automated/linux/iperf/